go/sqltypes/proto3_test.go

```diff
@@ -244,3 +244,80 @@ func TestQueryReponses(t *testing.T) {
 		t.Errorf("reverse:\n%#v, want\n%#v", reverse, queryResponses)
 	}
 }
+
+func TestProto3ValuesEqual(t *testing.T) {
+	for _, tc := range []struct {
+		v1, v2   []*querypb.Value
+		expected bool
+	}{
+		{
+			v1: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+			},
+			v2: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+				{
+					Type:  1,
+					Value: []byte{0, 1, 2},
+				},
+			},
+			expected: false,
+		},
+		{
+			v1: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+				{
+					Type:  1,
+					Value: []byte{0, 1, 2},
+				},
+			},
+			v2: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+				{
+					Type:  1,
+					Value: []byte{0, 1, 2},
+				},
+			},
+			expected: true,
+		},
+		{
+			v1: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+				{
+					Type:  1,
+					Value: []byte{0, 1},
+				},
+			},
+			v2: []*querypb.Value{
+				{
+					Type:  0,
+					Value: []byte{0, 1},
+				},
+				{
+					Type:  1,
+					Value: []byte{0, 1, 2},
+				},
+			},
+			expected: false,
+		},
+	} {
+		if got := Proto3ValuesEqual(tc.v1, tc.v2); got != tc.expected {
+			t.Errorf("expected %v, but got %v", tc.expected, got)
+		}
```

Comment:
change this to `require.Equal(t, tc.expected, Proto3ValuesEqual(tc.v1, tc.v2))`

Reply:
I have changed it. PTAL.